pkg/apis/kops/componentconfig.go

```diff
@@ -380,6 +380,8 @@ type KubeAPIServerConfig struct {
 	RuntimeConfig map[string]string `json:"runtimeConfig,omitempty" flag:"runtime-config"`
 
 	AnonymousAuth *bool `json:"anonymousAuth,omitempty" flag:"anonymous-auth"`
+	
+	KubeletPreferredAddressTypes []string `json:"kubeletPreferredAddressTypes,omitempty" flag:"kubelet-preferred-address-types"`
```

Comment:
Probably not worth it for these - they directly map to the flags.  Eventually this will be part of the k8s API, and we'll be able to delete this one.